lisa/tools/df.py

```diff
@@ -25,6 +25,26 @@ def command(self) -> str:
     def can_install(self) -> bool:
         return True
 
+    def get_mount_info_for_dir(
+        self, directory: str, force_run: bool = False
+    ) -> Optional[PartitionInfo]:
+        out = self.run(
+            parameters=f"{directory}", force_run=force_run, shell=True, sudo=True
+        ).stdout
+        df_entries = find_patterns_groups_in_lines(out, [self._DF_ENTRY_REGEX])[0]
+        if len(df_entries) == 0:
+            self._log.warn(f"Path: {directory} not exist!")
```

Comment:
Don't print warn line. It's not helpful for troubleshooting. Raise an exception, and we can know which images need to be fixed.

Reply:
fixed

Reply:
Haven't fixed.